docs/developer-env.md

```diff
@@ -0,0 +1,530 @@
+
+:::{note}
+Conda environments can lead to inconsistencies across systems due to dependency resolution and OS variations.
+:::
+
+The preferred method for installing Conda is through Miniconda, a lightweight version of Anaconda that includes Conda and its dependencies.
+
+````{tabs}
+
+```{group-tab} Windows
+
+```
+
+```{group-tab} macOS
+
+To install Conda on macOS:
+
+1. Visit the [Miniconda](https://docs.anaconda.com/miniconda/#miniconda) website.
+1. Download the latest version of the `.pkg` Miniconda installer. 
+1. Double-click the `.pkg` file.
+1. Follow the step-by-step setup installation instructions.
+1. When the installation finishes, open a new terminal window and run `conda list` to verify Conda was installed correctly.
+
+See [Quick command line install](https://docs.anaconda.com/miniconda/#quick-command-line-install) for command line installation instructions.
+
+```
+
+```{group-tab} Linux
+
+To install Conda on Linux Debian/Ubuntu distributions:
+
+1. Visit the [Miniconda](https://docs.anaconda.com/miniconda/#miniconda) website.
+1. Download the latest version of the `.sh` Miniconda installer.
+1. In your terminal, run `bash <file name>.sh`, replacing `<file name>` with the installer file name.
+1. Follow the step-by-step setup installation prompts.
+1. When the installation finishes, open a new terminal window and run `conda list` to verify Conda was installed correctly.
+
+```
+
+````
+--->
+
+## Git
+
+Git provides powerful version control that helps track code changes. Git operates locally, meaning you don't need an internet connection to track changes, but it can also be used with remote platforms like GitHub, GitLab, or Bitbucket for collaborative development.
+
+Nextflow seamlessly integrates with Git for source code management providers for managing pipelines as version-controlled Git repositories.
+
+````{tabs}
+
+```{group-tab} Windows
+
+Git is already installed on most WSL distributions. You can check if it is already installed by running `git version`.
+
+To install the latest stable Git version on Linux Debian/Ubuntu distributions:
+
+1. Open a terminal window and run `sudo apt-get install git-all`.
+1. Once complete, run `git version` to verify Git was installed.
+
+See [git-scm documentation](https://git-scm.com/downloads/linux) for more information about installing Git on other Linux distributions.
+
+```
+
+```{group-tab} macOS
+
+To install Git on macOS with [Homebrew](https://docs.brew.sh/):
+
+1. Open a terminal window and run `brew install git`.
+
+   :::{note}
+   You must have Homebrew installed. See [Homebrew installation](https://docs.brew.sh/Installation) for instructions.
+   :::
+
+1. Once complete, run `git version` to verify Git was installed.
+
+To install Git on macOS with [Xcode](https://developer.apple.com/xcode/):
+
+1. Open the App Store on your Mac.
+1. Sign in to your Apple Account.
```

Comment:
Technically the user should already be signed in but if not, I'd move this to step 1.

Reply:
Manually removed